[2025-03-05T09:06:08.373+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T09:06:08.393+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-05T09:05:56.011253+00:00 [queued]>
[2025-03-05T09:06:08.402+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-05T09:05:56.011253+00:00 [queued]>
[2025-03-05T09:06:08.403+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-05T09:06:08.416+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): training_basic_classifier> on 2025-03-05 09:05:56.011253+00:00
[2025-03-05T09:06:08.427+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=480) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-05T09:06:08.429+0000] {standard_task_runner.py:72} INFO - Started process 493 to run task
[2025-03-05T09:06:08.429+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline_demo', 'training_basic_classifier', 'manual__2025-03-05T09:05:56.011253+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmp9g0smg6g']
[2025-03-05T09:06:08.430+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask training_basic_classifier
[2025-03-05T09:06:08.476+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-05T09:05:56.011253+00:00 [running]> on host aacdd75e5d47
[2025-03-05T09:06:08.549+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ml_pipeline_demo' AIRFLOW_CTX_TASK_ID='training_basic_classifier' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T09:05:56.011253+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T09:05:56.011253+00:00'
[2025-03-05T09:06:08.550+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-05T09:06:08.551+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-05T09:06:08.551+0000] {logging_mixin.py:190} INFO - Current task name:training_basic_classifier state:running start_date:2025-03-05 09:06:08.394211+00:00
[2025-03-05T09:06:08.551+0000] {logging_mixin.py:190} INFO - Dag name:ml_pipeline_demo and current dag run status:running
[2025-03-05T09:06:08.552+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-05T09:06:08.557+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/sklearn/utils/validation.py:1408: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)

[2025-03-05T09:06:08.663+0000] {logging_mixin.py:190} INFO - ML model training done and saved to /mlmodel.pkl
[2025-03-05T09:06:08.664+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T09:06:08.677+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-05T09:06:08.678+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ml_pipeline_demo, task_id=training_basic_classifier, run_id=manual__2025-03-05T09:05:56.011253+00:00, execution_date=20250305T090556, start_date=20250305T090608, end_date=20250305T090608
[2025-03-05T09:06:08.712+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-05T09:06:08.713+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-05T09:06:08.714+0000] {logging_mixin.py:190} INFO - Dag name:ml_pipeline_demo queued_at:2025-03-05 09:05:56.034225+00:00
[2025-03-05T09:06:08.714+0000] {logging_mixin.py:190} INFO - Task hostname:aacdd75e5d47 operator:PythonOperator
[2025-03-05T09:06:08.766+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T09:06:08.815+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T09:06:08.819+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
